PWGHF/TableProducer/candidateCreatorXic0Omegac0.cxx

```diff
@@ -958,12 +958,12 @@ struct HfCandidateCreatorXic0Omegac0 {
       auto charmbaryonChi2OverNdf = kfOmegac0Candidate.chi2GeoOmegac / charmbaryonNDF;
 
       kfOmegac0Candidate.chi2MassV0 = kfV0MassConstrained.GetChi2();
-      auto v0NDF_m = kfV0MassConstrained.GetNDF();
-      auto v0Chi2OverNdf_m = kfOmegac0Candidate.chi2MassV0 / v0NDF_m;
+      auto v0Ndf_m = kfV0MassConstrained.GetNDF();
```

Comment:
This does not make it `lowerCamelCase` (which does not allow underscores).

Reply:
Thanks for your comment. I have made the changes. Could you approve it if everything looks okay?